services/cloudapi/state.go

```diff
@@ -18,8 +19,8 @@ func GetStateChangeConf(meta interface{}, d *schema.ResourceData, location strin
 		Refresh:        resourceStateRefreshFunc(meta, location),
 		Timeout:        d.Timeout(timeoutType),
 		MinTimeout:     5 * time.Second,
-		Delay:          0,   // Don't delay the start
-		NotFoundChecks: 600, //Setting high number, to support long timeouts
+		Delay:          0,    // Don't delay the start
+		NotFoundChecks: 9999, //Setting high number, to support long timeouts
```

Comment:
@cristiGuranIonos the title contains `for k8s large clusters` but it seems like the change affects all components that use cloud API service, is this intended? shouldn't we do this only for k8s clusters?

Reply:
This only affects when setting a really high timeout. So even if practically it will only get used for k8s(because nothing else takes this long to create), it makes sense to change it for all cloudapi resources.